org.gnome.Cheese.yml

```diff
@@ -0,0 +1,75 @@
+id: org.gnome.Cheese
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: 3.34
```

Comment:
This should be a string instead.

Reply:
Should be fixed in https://github.com/flathub/flathub/pull/1320/commits/b185df4fa279eb0bfef9ad620b8e76bf6d70d42a Thanks.